alpine/entrypoint/post_start.sh

```diff
@@ -5,47 +5,13 @@ do
    [[ "${LOGLINE}" == *"Binding bundle: [landing-page"* ]] && pkill -P $$ tail
 done
 
-if [ -n "$INSTALL_FEATURES" ]; then
-  if [[ $INSTALL_FEATURES == *";"* ]]; then
-  _featureCount=$[$(echo $INSTALL_FEATURES | grep -o ";" | wc -l) + 1]
-  else
-    _featureCount=1
-  fi
+$APP_HOME/bin/client waitForReady -r 12 -d 10
+$APP_HOME/bin/client admin:configure /opt/ddfConfig.json -r 12 -d 10
```

Comment:
What happens if a config file isn't provided by a downstream container or in a volume mount? Should this be wrapped in a check for an env var pointing to the file in case of a container run that isn't using an automated install?